scenarios/evaluate/simulate_index/simulate_input_index.ipynb

```diff
@@ -0,0 +1,376 @@
+    "    return {\"messages\": messages[\"messages\"], \"stream\": stream, \"session_state\": session_state, \"context\": context}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Generate Query Responses from index\n",
+    "In this example we use a search index as raw text generate Query Response pairs. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "\n",
+    "\n",
+    "def generate_text_from_index(search_term: str) -> str:\n",
+    "    url = f\"{search_endpoint}/indexes/{index_name}/docs/search?api-version=2024-07-01\"\n",
+    "    headers = {\"api-key\": search_api_key, \"Content-Type\": \"application/json\"}\n",
+    "    search_query = {\"search\": search_term, \"top\": 10}\n",
+    "    response = requests.post(url=url, headers=headers, data=json.dumps(search_query))\n",
+    "\n",
+    "    text = \"\"\n",
+    "    if response.status_code == 200:\n",
+    "        results = response.json()\n",
+    "        for result in results[\"value\"]:\n",
+    "            text += result[\"description\"]\n",
+    "\n",
+    "    return text[:5000]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "real_estate_search_term = \"New York\"\n",
+    "text = generate_text_from_index(real_estate_search_term)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Call to simulator\n",
+    "This call to the simulator generates 4 query response pairs in its first pass.\n",
+    "In the second pass, it picks up one task, pairs it with a query (generated in previous pass) and sends it to the configured llm to build the first user turn. This user turn is then passed to the `callback` method. The conversation continutes till the `max_conversation_turns` turns.\n",
+    "\n",
+    "The output of the simulator will have the original task, original query, the original query and the response generated from the first turn as expected response. You can find them in the `context` key of the conversation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "outputs = await simulator(\n",
+    "    target=callback,\n",
+    "    text=text,\n",
+    "    num_queries=4,\n",
+    "    max_conversation_turns=3,\n",
+    "    tasks=[\n",
+    "        f\"I am a prospective buyer and I want to learn more about {real_estate_search_term}\",\n",
+    "        f\"I am a real estate agent and I want to inform potential buyers about {real_estate_search_term}\",\n",
+    "        f\"I am a researcher and I want to do a detailed research on {real_estate_search_term}\",\n",
+    "        f\"I am a statistician and I want to do a detailed table of factual data concerning {real_estate_search_term}\",\n",
+    "    ],\n",
+    ")"
```

Comment:
Maybe add another cell to show some advanced use for the simulator?
We can also make this change in the next PR - https://github.com/Azure-Samples/azureai-samples/pull/135/files#diff-3c4bd020ea01a97cceed3b03cd8719a64d4729c2506f9208dd6cdf6cb3a248ad